app/src/actions/user.js

```diff
@@ -57,7 +57,7 @@ export function logout() {
       type: LOGOUT
     });
     window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
-    if (window.location.pathname.match('^\/map')) {
+    if (window.location.pathname.match('/map')) {
```

Comment:
I don't have that warning. Could you please run `./node_modules/.bin/eslint -c .eslintrc app/src/actions/user.js` and tell me what gives?